src/features/fee-row/components/transaction-fee.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { AuthType } from '@stacks/transactions';
+
+import { stacksValue } from '@common/stacks-utils';
+import { useTxForSettingsState } from '@store/transactions/transaction.hooks';
+import { useFeeState } from '@store/transactions/fees.hooks';
+
+export function TransactionFee(): JSX.Element | null {
+  // TODO: Replace using this (note by atom)
```

Comment:
Use deprecated tsdoc if needed

Reply:
I don't know what this comment means?

Reply:
If it's really to be replaced, you could add the `/** @deprecated */` comment, and IDE's will interpret it (and help stop further use)